.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: php
+sudo: false
+
+before_install:
+  - travis_retry composer self-update
```

Comment:
As I understand it, travis does this now automatically, so not really necessary to do in the build anymore.

>Note that we update composer every time we update the PHP build environment, which is every 30-60 days. 

https://docs.travis-ci.com/user/languages/php/